modules/amp-360/pages/amp-360-plugin.adoc

```diff
@@ -0,0 +1,44 @@
+:current-module: amp-360
+include::ROOT:partial$amp-gobal-attributes.adoc[]
+:imagesdir: image-catalog
+:amp-plugin-sample-ios: https://mdtp-a.akamaihd.net/amp-ios-sdk/samples/AmpVR360Sample.zip
+:plugin-common-name:360
+
+= Amp 360
+
+The Amp 360 plugin allows you to play corretly content formatted for virtual reality purposes.
+
+You can check out our basic implementation, please refer to our application samples:
+
+* {amp-plugin-sample-ios}[AMP {plugin-common-name} application sample for iOS].
```

Comment:
@mchacong  
Miguel,
what is the link, or indication for the user to find/download corresponding samples.?